### ISSUE TITLE
bug: SOS emergency link missing from mobile navigation menu

### ISSUE THREAD

**Issue by nishupr** [Shaurya1907/PYSCHECARE-main#231]
## Description
The desktop navigation includes an SOS emergency link but it is 
completely absent from the mobile navigation menu. On a mental health 
platform, the SOS feature must be accessible on ALL screen sizes.

## Location
`index.html`

Desktop nav has:
```html
<li><a href="otherHTML/sos.html" class="sos-link">
  <i class="fas fa-exclamation-circle"></i> <span>SOS</span>
</a></li>
```

Mobile nav is missing it entirely.

## Fix
Add the SOS link to the mobile nav:
```html
<li><a href="otherHTML/sos.html" class="mobile-view-list sos-link">
  <i class="fas fa-exclamation-circle" aria-hidden="true"></i> SOS
</a></li>
```
